LayoutKit/Internal/TextExtension.swift

```diff
@@ -0,0 +1,81 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+
+/// DO NOT move this file to `Text.swift`
+/// because this extension is using `UIKit` and serves for
+/// `LabelLayout` and `TextLayout`
+extension Text {
```

Comment:
I don't remember if Text.swift is actually used in the macOS build. I might have only done that because it didn't require UIKit. If Text.swift isn't actually used anywhere else, I would just move this to Text.swift and remove Text.swift from the macOS target.